devices/SNS-4/out/errors.json

```diff
@@ -0,0 +1,5 @@
+Exceptions for SNS-4
+  While validating envelope
+    system.physical_tag.asset.site ZZ-TRI-FECTA does not match expected zregistry1
+  devices/SNS-4
```

Comment:
This doesn't seem right -- do you know why these errors/exceptions are being introduced?